lib/rubygems/defaults.rb

```diff
@@ -45,6 +45,27 @@ def self.default_dir
     @default_dir ||= File.join(*path)
   end
 
+  ##
+  # Directory to StdLib default gems
+
+  def self.default_gems_dir
+    path = if defined? RUBY_FRAMEWORK_VERSION
+             [
+               File.dirname(RbConfig::CONFIG['sitedir']),
+               'Default',
```

Comment:
Is the capital `D` intentional?

Reply:
Yes, similarly to `Gems` in `.default_dir` above. But this was refactored and dropped, so there is no point to discuss this anymore.